spec/index.html

```diff
@@ -1142,66 +1142,45 @@ <h2>Best Practices</h2>
         -->
       </pre>
 
-      <p>The need to quote these keywords has to be learnt, and introduces one more little irregularity to the document
+      <p>
+        The need to quote these keywords has to be learnt, and introduces one more little irregularity to the document
         author's life. Further, on most keyboard layouts, typing quotes will require `Shift`, which reduces typing speed,
-        albeit slightly.</p>
+        albeit slightly.
+      </p>
+
+      <p>
+        In order to avoid this, the context might introduce custom mappings for JSON-LD keywords —
+        to make authoring more convenient. The exact mapping might vary depending on the domain, but we provide two
+        examples, both published at <a href="https://json-ld.org">json-ld.org</a>:
+      </p>
 
-      <p>In order to avoid this, the context might introduce custom mappings for the necessary keywords. For instance,
-        [[schema-org]] context redefines `@id` as just `id` — which seems to be much more convenient to type, and
-        no more difficult to remember.</p>
+      <ul>
+        <li>
+          <a href="https://json-ld.org/contexts/convenience.jsonld"><code>convenience.jsonld</code></a>
+          maps <code>@id</code> → <code>id</code> and so on, removing the <code>@</code> character
+        </li>
+        <li>
+          <a href="https://json-ld.org/contexts/dollar-convenience.jsonld"><code>$convenience.jsonld</code></a>
```

Comment:
Should probably be named based on the actual filename.

```suggestion
          <a href="https://json-ld.org/contexts/dollar-convenience.jsonld"><code> dollar-convenience.jsonld </code></a>
```